enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1334 @@
+   provide maintenance windows for customers.
+1. Deliver a consistent change management experience across all platforms and profiles (e.g. Standalone, ROSA, HCP).
+1. Enable SRE to, when appropriate, make configuration changes on a customer cluster and have that change actually 
+   take effect only when permitted by the customer's change management preferences.
+1. Do not subvert expectations of customers well served by the existing fully self-managed cluster update.
+1. Ensure the architectural space for enabling different change management strategies in the future.
+1. Directly support the strategic separation of control-plane and worker-node update phases by empowering cluster 
+   lifecycle administrators with change management strategies that provide them fine-grained control over exactly
+   when and how worker-nodes are updated to a desired configuration even if no regular maintenance schedule is possible.
+   
+### Non-Goals
+
+1. Allowing control-plane upgrades to be paused midway through an update. Control-plane updates are relatively rapid 
+   and pausing will introduce unnecessary complexity and risk. 
+1. Requiring the use of maintenance schedules for OpenShift upgrades (the changes should be compatible with various 
+   upgrade methodologies – including being manually triggered).
+1. Allowing Standalone worker-nodes to upgrade to a different payload version than the control-plane 
+   (this is supported in HCP, but is not a goal for standalone).
+1. Exposing maintenance schedule controls from the oc CLI. This may be a future goal but is not required by this enhancement.
+1. Providing strict promises around the exact timing of upgrade processes. Maintenance schedules will be honored to a 
+   reasonable extent (e.g. upgrade actions will only be initiated during a window), but long-running operations may 
+   exceed the configured end of a maintenance schedule.
+1. Implementing logic to defend against impractical maintenance schedules (e.g. if a customer configures a 1-second 
+   maintenance schedule every year). Service Delivery may want to implement such logic to ensure upgrade progress can 
+   be made.
+1. Automatically initiating updates to `ClusterVersion`. This will still occur through external actors/orchestration. 
+   Maintenance schedules simply give the assurance that changes to `ClusterVersion` will not result in material changes 
+   until permitted by the defined maintenance schedules.
+
+## Proposal
+
+### Change Management Overview
+Establish a new namespaced Custom Resource Definition (CRD) called `ChangeManagementPolicy` which allows cluster lifecycle
+administrators to capture their requirements for when resource(s) associated with the policy can initiate material 
+changes on the cluster.
+
+Add a `changeManagement` stanza to several existing resources in the OpenShift ecosystem which can reference
+the new `ChangeManagementPolicy` resource to restrict when and how their associated controllers can initiate

```

Comment:
Described a bit later - manageable resources can refer to only one policy. Permitting multiple means returning to the non-deterministic calculation of overlapping intervals. Something I'm happy to avoid with a single policy.